DirectEmployers/MyJobs
# [PD 2808] loading in all charts for display

This task was to load in the charts for display. The charts are switching depending on the data coming in, however the data is current dumbie data that is being supplied. The actual data will go back in once the API is complete for chart switching in which I will have to make some modifications to the logic behind this. 

To Test:
Pull down the repository and open analytics. You WILL need to run an NPM install as the packages have been changed and updated. You can filter down inside of the table and see the charts switch depending on which data is present from the Maps to the Bar Chart.
